python/foglamp/plugins/south/cc2650/cc2650.py

```diff
@@ -0,0 +1,403 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Module for Sensortag CC2650 plugin """
+
+import copy
+import json
+import uuid
+import threading
+
+from foglamp.common import logger
+from foglamp.plugins.common import utils
+from foglamp.plugins.south.cc2650.sensortag_cc2650 import *
+from foglamp.services.south import exceptions
+
+__author__ = "Amarendra K Sinha"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+_DEFAULT_CONFIG = {
+    'plugin': {
+         'description': 'TI SensorTag South Plugin',
+         'type': 'string',
+         'default': 'cc2650'
+    },
+    'assetPrefix': {
+        'description': 'Asset prefix',
+        'type': 'string',
+        'default': 'CC2650_%M_'
```

Comment:
should have optional parameter order with value 2

Reply:
thanks for replacing `/` with `_` as  OCS does not accept `/` even in post payload. may be  PI Server too. may be we need `-` hyphen here. as long as its configurable, NOT a big deal